README.md

```diff
@@ -333,6 +361,260 @@ This may take a few minutes to finish.
+    "results": {
+      "id_predicate":       "http://purl.org/dc/terms/identifier",
+      "label_predicate":    "http://www.w3.org/2004/02/skos/core#prefLabel",
+      "altlabel_predicate": "http://www.w3.org/2004/02/skos/core#altLabel",
+      "sameas_predicate":   "http://schema.org/sameAs"
+    }
+  },
+  "search": {
+    "url": {
+      "@context": "http://www.w3.org/ns/hydra/context.jsonld",
+      "@type": "IriTemplate",
+      "template": "http://experimental.worldcat.org/fast/search?query={?subauth}+all+%22{?query}%22&sortKeys=usage&maximumRecords={?maximumRecords}",
+      "variableRepresentation": "BasicRepresentation",
+      "mapping": [
+        {
+          "@type": "IriTemplateMapping",
+          "variable": "query",
+          "property": "hydra:freetextQuery",
+          "required": true
+        },
+        {
+          "@type": "IriTemplateMapping",
+          "variable": "subauth",
+          "property": "hydra:freetextQuery",
+          "required": false,
+          "default": "cql.any"
+        },
+        {
+          "@type": "IriTemplateMapping",
+          "variable": "maximumRecords",
+          "property": "hydra:freetextQuery",
+          "required": false,
+          "default": "20"
+        }
+      ]
+    },
+    "qa_replacement_patterns": {
+      "query":   "query",
+      "subauth": "subauth"
+    },
+    "language": ["en"]
+    "results": {
+      "id_predicate":       "http://purl.org/dc/terms/identifier",
+      "label_predicate":    "http://www.w3.org/2004/02/skos/core#prefLabel",
+      "sort_predicate":     "http://www.w3.org/2004/02/skos/core#prefLabel"
+    },
+    "subauthorities": {
+      "topic":          "oclc.topic",
+      "geographic":     "oclc.geographic",
+      "event_name":     "oclc.eventName",
+      "personal_name":  "oclc.personalName",
+      "corporate_name": "oclc.corporateName",
+      "uniform_title":  "oclc.uniformTitle",
+      "period":         "oclc.period",
+      "form":           "oclc.form",
+      "alt_lc":         "oclc.altlc"
+    }
+  }
+}
+```
+
+NOTES:
+* term: (optional) is used to define how to request term information from the authority and how to interpret results.
+  * url: (required) templated link representation of the authority API URL and mapping of parameters for requesting term information from the authority
+    * template: is the authority API URL with placeholders for substitution parameters in the form {?var_name}
+      * NOTE: {?term_id} (required) and {?subauth} (optional) are expected to match to QA params (see qa_replacement_patterns to match QA params with mapping variables)
+      * Additional substitutions can be made in the authority API if supported by the authority by adding additional mappings.  Search has an example with maximumRecords.
+        * variable: should match a replacement pattern in the template  (e.g. variable: maximumRecords  ==>  {?maximumRecords}
+        * required: true | false  (NOTE: Not enforced at this time.)
```

Comment:
I'd like to open an issue for this after the PR is merged.  There are a few system required substitutions that are checked for.  If a config adds its own substitution and marks it required, there is no check for that.  The required marker is part of the templated URLs spec which is why it is there.  I agree that if it is there, it should be checked.